web/locales/en/messages.ftl

```diff
@@ -47,8 +47,8 @@ sc-howto-addsen-post-1 = We prefer natural/conversational sentences. While phone
 sc-howto-cite-title = How to Cite
 sc-howto-cite-pre-1 = It's important that you reference where you found the public licence to avoid plagiarism and to allow for follow ups on <copyrightIssuesLink>copyright issues</copyrightIssuesLink>. Here are a few ways you can cite the work.
 sc-howto-cite-item-1 = You could include the website, e.g "Common Voice - https://commonvoice.mozilla.org/"
-sc-howto-cite-item-2 = You could use Academic referencing style, e.g harvard style "Mozilla (2021) Common Voice. Available at https://commonvoice.mozilla.org/ (Accessed: 15th September 2021)"
-sc-howto-cite-item-3 = For public licence text not available online, you could use Academic referencing style e.g harvard style "Jess (2021) My Public licence poems"
+sc-howto-cite-item-2 = You could use academic referencing style, e.g Harvard style "Mozilla (2021) Common Voice. Available at https://commonvoice.mozilla.org/ (Accessed: 15th September 2021)"
+sc-howto-cite-item-3 = For public licenced text not available online, you could use academic referencing style e.g Harvard style "Jess (2021) My Public licence poems"
```

Comment:
Without being a lawyer, editor or copywriter, I think "public license text" would be better than your suggestion. I feel "licenced" introduces some ambiguity. Maybe "For public domain texts not available online .." would be more descriptive?

Reply:
Changed to "public domain text"...